src/Transports.AspNetCore/GraphQLHttpMiddlewareOptions.cs

```diff
@@ -1,3 +1,5 @@
+using MediaTypeHeaderValueMs = Microsoft.Net.Http.Headers.MediaTypeHeaderValue;
```

Comment:
Just to shorten line 103 ?

Reply:
There are two different `MediaTypeHeaderValue` classes both within scope.